orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/pipeline/ManualJudgmentStage.groovy

```diff
@@ -155,9 +156,25 @@ class ManualJudgmentStage implements StageDefinitionBuilder, RestartableStage, A
           application: stage.execution.application
         ),
         additionalContext: [
+          stageName: stage.name,
+          stageId: stage.refId,
+          restrictExecutionDuringTimeWindow: stage.context.restrictExecutionDuringTimeWindow,
           instructions: stage.context.instructions ?: ""
         ]
-      ))
+      )
+
+      if (stage.execution instanceof Pipeline) {
```

Comment:
in other places we send the entire execution. Might make sense to do that as well here so that if we decided to improve templating in the future manual judgment is not a special snowflake